index.html

```diff
@@ -1,72 +1,82 @@
-                  What I miss: My son loves this museum, and I miss the time I could spend here with him.
-              </div>
-              <div class="link">
-              <a href="https://goo.gl/maps/WvBUTQBVMSymd4UCA" target="_blank">location</a>
-              </div>
-          </section>
-          
-         
-        
-          
-          
-      </div>
-       
-  <footer>
-      made by Sergei Golovnev for REDI: Frontend Development 2
-      </footer>
-    </body>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <title>My ReDi Project</title>
+    <link rel="stylesheet" href="stylesheet.css" type="text/css">
+</head>
+<body>
+    <header>
+        <ul>
+            <li><a id="menu_burger"><img src="burger_button_w.png" height="15px"></a></li>
+            <li><a href="">HOME</a></li>
+            <li><a href="">ABOUT ME</a></li>
+            <li><a href="index.html" class="current_page">BASE PAGE</a></li>
+        </ul>
+    </header>
+    <div class="main">
+        <section class="post">
+            <h2 class="post_title">Traveling in Bavaria</h2>
+            <div>
+                <img class="post_pic" src="travel.jpg">
+            </div>
+            <div class="post_description">
+                <p>What I miss: I love traveling and miss the nature and landscapes that I see while traveling.</p>
+            </div>
+            <div class="post_link">
+                <a class="map_link" href="https://goo.gl/maps/Vx9rXPP3NdG63ysq7" target="_blank">location</a>
+            </div>
+        </section>
+        <section class="post">
+            <h2 class="post_title">Tierpark Hellabrunn</h2>
+            <div>
+                <img class="post_pic" src="tierpark.jpg" width=100%>
+            </div>
+            <div class="post_description">
+                <p>What I miss: I miss the moments we spent there with my family. Our tickets will end on April 16 :(</p>
+            </div>
+            <div class="post_link">
+                <a class="map_link" href="https://goo.gl/maps/XMUj6szEoj6F7ufn6" target="_blank">location</a>
+            </div>
+        </section>
+        <section class="post">
+            <h2 class="post_title">Deutsches Museum Verkehrszentrum</h2>
+            <div>
+                <img class="post_pic" src="museum.jpg" width=100%>
+            </div>
+            <div class="post_description">
+                <p>What I miss: My son loves this museum, and I miss the time I could spend here with him.</p>
+            </div>
+            <div class="post_link">
+                <a class="map_link" href="https://goo.gl/maps/WvBUTQBVMSymd4UCA" target="_blank">location</a>
+            </div>
+        </section>
+    </div>
+    <footer>
+        <p>Made by Sergei Golovnev for REDI: Frontend Development 2</p>
+    </footer>
+    <script>
+        let menuBurgerButton = document.getElementById("menu_burger");
+
+        function clickBurgerProcessor() {
```

Comment:
there is not official convention which I can suggest for this kind of functions. Since you want to handle users action then you can put it in the name like this  **handleBurgerClick** or **handleMenuClick**. Then it's a bit more clear what this function is doing. Because for me processor could do file processing, data processing and so on.

Reply:
- rename function clickBurgerProcessor to  handleMenuClick - fixed